### PR TITLE
framework/messaging : Add parsing the received packet based on its version

### PR DESCRIPTION
**Pull request by jeongchanKim**
1. framework/messaging : Add parsing the received packet based on its version
2. apps/elf : Change app stacksize and priority, Minimize the printed log.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [fb8ba6c60f3118e889653635442722223cb08877] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [fb8ba6c60f3118e889653635442722223cb08877] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [d5d601f24075e666e922ea78674dcafc6176c14f] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [d5d601f24075e666e922ea78674dcafc6176c14f] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [bda3c2d0fd45eac724912e6e4a5e8d7742523462] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [bda3c2d0fd45eac724912e6e4a5e8d7742523462] - Code Rule Check OK.

File: framework/src/messaging/messaging_common.c [hunk 1]
```diff
@@ -103,6 +107,50 @@ int messaging_set_notify_signal(int signo, _sa_sigaction_t func)
...
+{
+	int my_version;
+	int msg_version;
+	int parsing_version;
+	int ret = OK;
+	int offset;
+
+	my_version = messaging_get_version();
+
+	/* Received packet(version 1) is like below.
```
Comment (Taejun-Kwon) [inline framework/src/messaging/messaging_common.c]: If the version 1 should follow below type, you had better define structure.
```
struct packet_header_s {
     uint32_t version;
     uint32_t offset;
     uint32_t sender_pid;
     uint32_t msg_type;
}
...
struct message_packet {
    struct packet_header_s header;
    data....
}
....
#define offsetof(type, member) ((size_t)&(((type *)0)->member))
...
case 1 :
         memcpy(sender_pid, offsetof(struct packet_header_s, sender_pid, sizeof(pid_t));
...
```

**Comment by seinfra** [comment]
Target : [35a674fcdb8621e8b2b70eaa98fd9bcff372c512] - Code Rule Check OK.

File: framework/src/messaging/messaging_common.c (hunk 1 above)
Comment (jeongchanKim) [reply framework/src/messaging/messaging_common.c]: Thanks. It is more clear.
I found that we already have offsetof in <stddef.h>.
I modified.

**Comment by seinfra** [comment]
Target : [35a674fcdb8621e8b2b70eaa98fd9bcff372c512] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [b5cd91ed000e8ae9137c7681aaf5414c6bda213f] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [b5cd91ed000e8ae9137c7681aaf5414c6bda213f] - Code Rule Check OK.

**Pull request merged**